### PR TITLE
[AV2-390] Customise the toast message

### PR DESCRIPTION
**Pull request by sasangachathumal**
Jira story - https://intersective.atlassian.net/browse/AV2-390
remove toast from topics and show toast only on assessment which have plus check.
and customize toast by adding primary color to background and adding tick to toast.
and need to change text of toast to 'assessment complete!'

### CODE REVIEW

File: src/app/shared/notification/notification.service.ts [hunk 1]
```diff
@@ -71,24 +77,43 @@ export class NotificationService {
...
-    let color = 'success';
-    if (!success) {
-      color = 'danger';
-    }
-    return this.customToast({
+  async presentToast(message, success = false, duration?, customOptions?: ToastOptions) {
+    let toastOptions: ToastOptions = {
       message: message,
-      duration: duration || 2000,
+      duration: duration || 200000,
```
Comment (trtshen) [inline src/app/shared/notification/notification.service.ts]: @sasangachathumal  why 3mins++ ? 😅

Comment (sasangachathumal) [reply src/app/shared/notification/notification.service.ts]: @trtshen i'm still workin on this that added for testing. without making duration long can't inspect and check styles of toast. and this is not ready for review or testing. i'm still working on it.

**Comment by Barry-Chen-Intersective** [comment]
there is an error on running the test of CustomToastComponent, see if we need to fix it.
https://qareport.practeraco.de/appv2_coverage/report_22-40-08-10-09-2019/result.html

File: src/app/shared/notification/notification.service.ts (hunk 1 above)
Comment (trtshen) [reply src/app/shared/notification/notification.service.ts]: okok, i thought it's final, it's alright then, thanks @sasangachathumal

**Comment by Barry-Chen-Intersective** [comment]
https://qareport.practeraco.de/appv2_coverage/report_34-43-03-20-09-2019/result.html

**Pull request merged**